src/grid-column.component.ts

```diff
@@ -86,3 +92,7 @@ export class GridColumnComponent implements OnInit {
       return _.get(data, columnName) as string;
   }
 }
+
+export interface IStyleCallback {
```

Comment:
Lets not use "I" convention for interface names. Also lets include Cell in the name, so CellStyleCallback?
We also have similar interface declared in grid-options.ts called RowStyleCallback, another option would be to combine those and move them to separate file with just StyleCallback name and share same interface.